### ISSUE TITLE
🚀 Auto-submit saved solution on due date

### ISSUE THREAD

**Issue by heichel** [ls1intum/ArTEMiS#402]
### Is your feature request related to a problem?
It's very disappointing having saved an almost-finished solution of a text or modeling excercise but forgetting to press the submit button and being graded with zero points.

### Describe the solution you'd like
The (latest) saved solution should be automatically submitted upon the due date, so that even unfinished solutions can be graded with partial points.

**Comment by krusche** [comment]
I understand that you are disappointed. We are working on an automatic submission, however please keep in mind that you should not rely on it. In all other exercise systems (e.g. Moodle), you also have to submit your work.

**Comment by DeRealMorgan** [comment]
I want to add to that:
I do not know the reasons behind not allowing the re-submission of text exercises but
an easy (and for me an equal solution) would be to allow multiple submissions of the exercises.

**Comment by jpbernius** [comment]
Automatic submissions of text (and modeling) exercises were recently released.

Can you elaborate on how multiple submissions would benefit students in the case of a missed deadline, @DeRealMorgan?

**Comment by DeRealMorgan** [comment]
Yes e.g. I missed the deadline as I did not want to submit my answer.
I have worked on the solution of the exercises a few days ago but I wanted to read through them again (-> I did not submit).
Today I saw that I missed the deadline (which is very unfortunate as I really answered the questions).

I am confused now: [#449](https://github.com/ls1intum/ArTEMiS/pull/449) If I understand that correctly, previously saved answers will be automatically submitted - maybe that is another bug then as I was looking though the exercises today and the "submit" stated "submit (due date reached)" or something.
